.github/workflows/ci.yml

```diff
@@ -226,6 +228,8 @@ jobs:
           conda list
       - name: Install HDF5
         run: sudo apt update && sudo apt install -y libhdf5-dev
+      - shell: bash -l {0}
+        run: python -m pytest modin/test/backends/base/test_internals.py --backend=${{ matrix.backend }}
```

Comment:
In other cases, we control the backend through an environment variable.

Why is `--backend` pytest option now used?

Reply:
This is the way how we're running tests for the base backend, there is no backend called "base", because `BaseQueryCompiler` is an abstract class. We dynamically generate "base" backend just to run tests, by replacing abstract methods of `BaseQueryCompiler` via `PandasQueryCompilers`s methods. This logic was introduced in #2047